config.py

```diff
@@ -135,6 +134,15 @@ def create_config(config_file=None):
         logger.warning("Waveshare display library not installed.")
         sys.exit("Exiting...")
 
+# Display - Waveshare 2.13 (D) is 212 * 104 pixels
+if "waveshare2in13d" in conf["atm"]["display"]:
```

Comment:
These options are mutually exclusive. I'm not sure how to install the library (maybe i did that wrong) but why not using `elif` constructs? This fixed my issue.

Reply:
That's correct. I took the normal if so as not to touch the papirus block. If the correct display is choose in the config.ini, there are no problems either.

Reply:
We can make the change with a separate PR

Reply:
I'm not sure i understand the approach. Why don't you want to touch the papirus block? This is a configuration procedure and there is some kind of fallback-mode via "waveshare" (as it's not 100% specified). However, that works with elif as well and these values are mutually exclusive. Using "if", not elif, makes it error-prone: I had issues with the normal if-block. Maybe i missed the point somewhere: Where is the description of the installation of the waveshare-library? Maybe i did that wrong.
But that should not prevent doing elifs?!

Reply:
If you look at the existing PRs, you will see the touch of the papirus block will results in additional merge conflicts. I would therefore prefer to have a new PR with the elif. Then we can choose the time when we approve the adjustment. I have no problems using the normal if.